lib/policies/load-balancing.js

```diff
@@ -99,7 +99,9 @@ RoundRobinPolicy.prototype.newQueryPlan = function (keyspace, executionOptions,
  * after the local nodes. In other words, this policy guarantees that no
  * host in a remote data center will be queried unless no host in the local
  * data center can be reached.
- * @param {?String} [localDc] local datacenter name.
+ * @param {?String} [localDc] Deprecated: local datacenter name.  This value is
```

Comment:
Thinking about this some more, I suspect maybe we shouldn't mark this deprecated.   One use case where you might want to use this is when using execution profiles and you want to have different requests going to different data centers.  Instead proposing:

> local datacenter name.  This value overrides the 'localDataCenter' Client option and is useful for cases where you have multiple execution profiles that you intend on using for routing requests to different data centers.

Reply:
I ended up undeprecating this, but let me know if I should re-evaluate.